scripts/exglobal_oceanice_products.py

```diff
@@ -19,8 +19,7 @@ def main():
 
     # Pull out all the configuration keys needed to run the rest of steps
     keys = ['HOMEgfs', 'DATA', 'current_cycle', 'RUN', 'NET',
-            'COM_OCEAN_HISTORY', 'COM_OCEAN_GRIB',
-            'COM_ICE_HISTORY', 'COM_ICE_GRIB',
+            f'COM_{component.upper()}_HISTORY', f'COM_{component.upper()}_GRIB'
```

Comment:
I think this should either be `config.COMPONENT` unless you want to pull from the `task_config`:

```suggestion
            f'COM_{oceanice.task_config.component.upper()}_HISTORY', f'COM_{oceanice.task_config.component.upper()}_GRIB'
```